src/flatConfigEditor.ts

```diff
@@ -164,12 +164,29 @@ export class FlatConfigEditor implements vscode.CustomTextEditorProvider {
     )
     const document = await vscode.workspace.openTextDocument(flatFileUri)
     const rawFlatYaml = document.getText()
-    const parsedConfig = parse(rawFlatYaml)
 
     const dirName = workspaceRootUri.path.substring(
       workspaceRootUri.path.lastIndexOf('/') + 1
     )
 
+    let gitRepo = ''
+    const gitConfigFileUri = vscode.Uri.joinPath(
```

Comment:
Any reason you're not using the Git API class that we previously extracted for this purpose?

Reply:
yess you're right! I didn't want to pull in an extra lib when it wasn't necessary but I had totally forgotten it was already there. will update!

Reply:
@Wattenberger No need to change, just wanted to understand your thought process here!

Reply:
nah updated! that was a great call